runtime/src/snapshot_utils.rs

```diff
@@ -268,10 +269,14 @@ pub fn archive_snapshot_package(
         .map_err(|e| SnapshotError::IoWithSource(e, "create archive path"))?;
 
     // Create the staging directories
+    let staging_dir_prefix = match snapshot_package.snapshot_type {
+        SnapshotType::FullSnapshot => TMP_FULL_SNAPSHOT_PREFIX,
+        SnapshotType::IncrementalSnapshot => TMP_INCREMENTAL_SNAPSHOT_PREFIX,
+    };
```

Comment:
could maybe be a method `to_prefix()` on `SnapshotType`

Reply:
Done!